sources/mk/macedonia-municipalities.hjson

```diff
@@ -0,0 +1,136 @@
+{
+    versions: '>=1'
+    type: http
+    production: true
+    wikidata: Q646793
+    note: Municipality of North Macedonia
+    data: https://sophox.org/regions/geojson.json
+    attribution: [
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
+        {
+            type: id
+            name: iso_3166_2
+            desc: ISO-3166-2 identifier
+        }
+        {
+            type: property
+            name: label_en
+            desc: Subdivision name (English)
+        }
+        {
+            type: property
+            name: label_mk
+            desc: Subdivision name (Macedonian)
+        }
+        {
+            type: property
+            name: label_sq
+            desc: Subdivision name (Albanian)
+        }
+    ]
+    query: {
+        sparql:
+            '''
+            #defaultView:MapRegions
+            SELECT
+            (if(bound(?id2),?id2,?id) as ?id)
+            ?iso_3166_2 ?label_en ?label_mk ?label_sq ?wkidataOSM
```

Comment:
I don't think we need this variable.

```suggestion
            ?iso_3166_2 ?label_en ?label_mk ?label_sq 
```

Reply:
Yeah I spotted that one on the last query I shared on the comments of the PR, thanks!!